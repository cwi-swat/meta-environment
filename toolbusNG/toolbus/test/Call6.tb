process A(S : str) is rec-msg(a) . printf("A has received message from %s\n", S)

process B(S : str) is rec-msg(b) . printf("B has received message from %s\n", S)

process Main(B : str) is B("Main") . printf("Main completed\n")

process Sender(Msg : term) is snd-msg(Msg)
 
toolbus(Main("B"), Sender(a), Sender(b), A("toolbus"))