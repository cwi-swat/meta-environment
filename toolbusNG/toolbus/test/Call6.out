A has received message from "toolbus"
B has received message from "Main"
Main completed
